src/style.css

```
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.stars_star {
+  background-repeat: no-repeat;
+  background-position: 50%;
+  background-color: #fff;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars_star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 > :nth-child(-n+1),
+.stars--2 > :nth-child(-n+2),
+.stars--3 > :nth-child(-n+3),
+.stars--4 > :nth-child(-n+4),
+.stars--5 > :nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--0 > :nth-child(n),
+.stars--1 > :nth-child(n+2),
+.stars--2 > :nth-child(n+3),
+.stars--3 > :nth-child(n+4),
+.stars--4 > :nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  justify-content: start;
+  align-items: center;
+  margin-right: 4px;
+  /* justify-content: space-between; */
+  /* align-items: center; */
+  /* margin-right: 4px; */
+}
```

```suggestion
.stars {
  display: flex;
  justify-content: start;
  align-items: center;
  margin-right: 4px;
}
```
Удаляй комментарии и используй базовый класс